scripts/gen-hsclient.js

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const assert = require('bsert');
+const path = require('path');
+const os = require('os');
+const util = require('util');
+const cp = require('child_process');
+const exec = util.promisify(cp.exec);
+const fs = require('bfile');
+const Config = require('bcfg');
+
+const ROOT = path.dirname(__dirname);
+const NAME = 'hs-client';
+const HSD_PKG = require(path.join(ROOT, 'package.json'));
+
+const REMOTE = 'git@github.com:handshake-org/hs-client.git';
+
+const INIT_HS_CLIENT_PKG = {
+  name: 'hs-client',
+  description: 'HSD node and wallet client',
+  keywords: [
+    'http',
+    'request',
+    'socket.io',
+    'websockets'
+  ],
+  main: './lib/client/index.js',
+  bin: {
+    'hsd-cli': './bin/hsd-cli',
+    'hsd-rpc': './bin/hsd-rpc',
+    'hsw-cli': './bin/hsw-cli',
+    'hsw-rpc': './bin/hsw-rpc'
+  },
+  engines: {
+    node: '>=8.0.0'
+  }
+};
+
+const INHERIT_PROPERTIES = [
+  'version',
+  'license',
+  'repository',
+  'homepage',
+  'bugs',
+  'author'
+];
+
+const DEPENDENCIES = [
+  'bcfg',
+  'bcurl',
+  'bsert'
+];
+
+const COPY_FILES = [
+  '.npmignore',
+  '.gitignore',
```

Comment:
hsd's gitignore is a superset of hs-client's (current) gitignore so not a problem, but you think it needs a separate hardcoded def here instead of copying hsd?

Reply:
It's good enough for the purposes. It's not meant to be developed separately. Even if we add `symlink` later, superset is enough.